packages/@okta/i18n/src/properties/login.properties

```diff
@@ -842,3 +846,12 @@ enroll.mdm.step3 = Follow the instructions in your browser to set up <span class
 
 ## safe mode
 oie.safe.mode.title = Authenticator set up unavailable
+
+## Self Service Registraiton Form
+oie.user.profile.lastname= Last name
+oie.user.profile.firstname= First name
+oie.user.profile.primary.email= Primary email
+
+## Terminal errors
+idx.session.expired = The session has expired.
```

Comment:
`oie.session.expired` ?

Reply:
this error key comes from backend as `idx.session.expired `.. Should we add it to the map ?

Reply:
leaning toward add to map. don't like to exposure another key prefix (`idx`)